### PR TITLE
Add support for Java 9 multi release jars #61

### PR DESCRIPTION
**Pull request by liamsharp**
- A property for library jar exclusions (libraryJarExclusion)

### CODE REVIEW

**Comment by FibreFoX** [comment]
technically LGTM, maybe the parameter-name could be something better. @wvengen what do you think about this new parameter/PR?

**Comment by wvengen** [comment]
@cmorty any objections?
@liamsharp Github shows that there may be an indentation mismatch. Could you please check that you're using the right indentation?

**Comment by wvengen** [comment]
@liamsharp ping?

**Comment by liamsharp** [comment]
@wvengen Whitespace fixed up, I've created a new pull request:

https://github.com/wvengen/proguard-maven-plugin/pull/75

File: src/main/java/com/github/wvengen/maven/proguard/ProGuardMojo.java [hunk 1]
```diff
@@ -691,6 +682,19 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 		}
 	}
 
+    private void addLibraryJar(ArrayList<String> args, ArrayList<File> libraryJars, File file)
+    {
+        if (putLibraryJarsInTempDir) {
+            libraryJars.add(file);
+        } else {
+            args.add("-libraryjars");
+        	    args.add(fileToString(file));
```
Comment (wvengen) [inline src/main/java/com/github/wvengen/maven/proguard/ProGuardMojo.java]: This still looks like an indentation mismatch, right?

File: src/main/java/com/github/wvengen/maven/proguard/ProGuardMojo.java [hunk 2]
```diff
@@ -123,7 +123,13 @@
 	 * @parameter default-value="false"
 	 */
 	private boolean putLibraryJarsInTempDir;
-
+	
+	/**
```
Comment (wvengen) [inline src/main/java/com/github/wvengen/maven/proguard/ProGuardMojo.java]: oops, also indentation mismatch

File: src/main/java/com/github/wvengen/maven/proguard/ProGuardMojo.java [hunk 3]
```diff
@@ -60,7 +60,7 @@
 
 public class ProGuardMojo extends AbstractMojo {
 
-	/**
+    /**
```
Comment (wvengen) [inline src/main/java/com/github/wvengen/maven/proguard/ProGuardMojo.java]: one more space?

**Comment by liamsharp** [comment]
@wvengen Created another pull request, only found 1 indentation issue:

https://github.com/wvengen/proguard-maven-plugin/pull/76

**Comment by lasselindqvist** [comment]
Closing this since the new fixes are in https://github.com/wvengen/proguard-maven-plugin/pull/76

**Pull request closed without merging**